### ISSUE TITLE
Allow for multiple output formats from Exomiser

### ISSUE THREAD

**Issue by yaseminbridges** [monarch-initiative/pheval.exomiser#59]
Currently, the HTML and JSON outputs are output by default - a field in the `config.yaml` should be added to allow for other raw output formats (TSV_GENE, TSV_VARIANT, VCF)

**Comment by matentzn** [comment]
Would this be an opportunity to make the outputs fully configurable, i.e. supress the HTML output to save space if you know you wont need it? Maybe not, but the 1 Terrabyte warning from @souzadevinicius scared me somewhat!

**Comment by souzadevinicius** [comment]
It will be very welcoming

**Comment by yaseminbridges** [comment]
Yes - we will still need the JSON for processing the results but you will be able to limit to just these or add other output formats if you wish
